### DISCUSSION TITLE
After a long time a new version of Logisim: V3.9.0

### DISCUSSION THREAD

**Discussion by BFH-ktt1** [logisim-evolution/logisim-evolution#2045] [Announcements]
A new release, V3.9.0.

**Comment by maehne** [comment]
Great News! The RPM release file seems to have got the wrong version number: 
`logisim-evolution-4.0.0dev-1.x86_64.rpm`

**Comment by juli-99** [comment]
As with every release, the references have to be updated.
#1534 contains a list with known references.
